src/kernels/static_composable_kernel/include/kernel_algorithm/static_kernel_reduction_kernel_simple_configurator.hpp

```diff
@@ -8,15 +8,15 @@ namespace ck {
 
 // The simple configurator does not consider the "Reduce_MultiBlock" method, since it is usually
 // called to do the second reduction after the first calling of a "Reduce_MultiBlock" reduction.
-template <index_t BlockSize, index_t warpSize>
+template <index_t BlockSize, index_t WarpSize>
 struct ReduceKernelSimpleConfigurator
 {
-    static constexpr index_t numWarpsPerBlock = BlockSize / warpSize;
+    static constexpr index_t numWarpsPerBlock = BlockSize / WarpSize;
```

Comment:
This doesn't seem to need a compile-time constant? The two member functions do a normal branch, which will get optimised if you use HIP's `warpSize`